src/erl.rs

```diff
@@ -891,19 +872,6 @@ fn incrementing_args_list(arity: usize) -> String {
         .collect()
 }
 
-fn external_fun(name: &str, module: &str, fun: &str, arity: usize) -> Document {
-    let chars: String = incrementing_args_list(arity);
-
-    atom(name.to_string())
-        .append(format!("({}) ->", chars))
-        .append(line())
-        .append(atom(module.to_string()))
-        .append(":")
-        .append(atom(fun.to_string()))
-        .append(format!("({}).", chars))
-        .nest(INDENT)
```

Comment:
I would like to continue to generate the function so that it could be used from Erlang, Elixir, etc.

Reply:
Ah ok, misunderstood you there. I'll bring this back.